docs/runbooks/KubeVirtVMIExcessiveMigrations.md

```diff
@@ -0,0 +1,11 @@
+# KubeVirtVMIExcessiveMigrations
+
+## Meaning
+The `KubeVirtVMIExcessiveMigrations` alert means that a VirtualMachineInstance has been live-migrated more than 12 times over a period of 24 hours.  
+This migration rate for a single VMI is significantly higher than a normal operation of KubeVirt, even if taking an upgrade into consideration.
+
+## Impact
+This Alert can suggest an issue in the cluster's underlying infrastructure, e.g. network disruptions, node resource shortage, etc. that causes one VMI or more to be migrated every two hours in average in one day.
```

Comment:
@orenc1 I think this is still part of the meaning. The impact will probably be that the VM will suffer from performance issues or something like that.